pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -106,6 +106,17 @@ class CustomPartitioningSpoke(NormalSpoke, StorageCheckHandler):
     # If the user enters a smaller size, the GUI changes it to this value
     MIN_SIZE_ENTRY = Size("1 MiB")
 
+    DESIRED_CAPACITY_ERROR = N_(
+        "Specify the Desired Capacity in whole or decimal numbers, with an appropriate unit.\n\n"
+        "Spaces separating digit groups are not allowed. Units consist of a decimal or binary "
+        "prefix, and optionally the letter B. Letter case does not matter for units. The default "
+        "unit used when units are left out is MiB.\n\n"
```

Comment:
I would change the last sentence to:
``The default unit is MiB if no letter is specified.``

Seems easier to understand to me.